### DISCUSSION TITLE
clear the visited projects(repositories)

### DISCUSSION THREAD

**Discussion by Nuurik** [community/community#43576] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

how can i clear the visited projects(repositories) from "search or jump to"
![Capture](https://user-images.githubusercontent.com/66960695/211160080-0036a346-d6bb-428c-aaf5-9e6ed8e62d1d.PNG)
delete buttun and clearing google chrome history doesn't work
![Capture1](https://user-images.githubusercontent.com/66960695/211160161-8de25595-cf13-415f-8d2e-16d1fb81416f.PNG)
clearing google chrome history doesn't work because after clearing chrome history this "YetiForceCompany" repository doesn't disappear
![Capture2](https://user-images.githubusercontent.com/66960695/211161122-dbec4647-fc21-4619-a2b1-293975726686.PNG)

please help, i have already searched this question in internet but i didn't find answer, mayby because my english level isn't high

**Comment by auchynnikau** [comment]
Looks like the only solution is to clear your browser history related to GitHub.

**Reply by Nuurik** [reply]
i tryed, all of them exept this "yetiforse" disappear
and thank you
